config.yaml

```diff
@@ -49,3 +49,14 @@ hardware:
 # Logging configuration
 logging:
   tensorboard_dir: logs/
+
+# Rendering configuration
+rendering:
+  render: True
```

Comment:
Do we need `render` flag? This is unwieldy. Why don't we set it as a mode option?